### ISSUE TITLE
tie ground truth labels to training data data manager object

### ISSUE THREAD

**Issue by aikramer2** [datascienceinc/Skater#192]
Current feature importance will break if using method='conditional-permutation' and  the size of the dataset is larger than the number of samples. This is because feature importance samples the input, but there's no corresponding sampling of the training labels. This could be solved by tying the two together in the same datamanager.

**Comment by aikramer2** [comment]
solved by https://github.com/datascienceinc/Skater/pull/198 closing
